app/src/main/java/com/tab/mybadges/ui/badgeList/BadgeListActivity.kt

```diff
@@ -0,0 +1,44 @@
+package com.tab.mybadges.ui.badgeList
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.lifecycle.Observer
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.tab.mybadges.MockBadgeListViewModel
+import com.tab.mybadges.R
+import com.tab.mybadges.databinding.ActivityBadgeListBinding
+import com.tab.mybadges.ui.userList.UserListAdapter
+
+class BadgeListActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityBadgeListBinding
+
+    private lateinit var recyclerViewBadgeList: RecyclerView
+    private lateinit var recyclerViewBadgeListAdapter: BadgeListAdapter
+    private lateinit var layoutManager: RecyclerView.LayoutManager
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityBadgeListBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        recyclerViewBadgeList = binding.rvBadgeList
+        recyclerViewBadgeListAdapter =
+            BadgeListAdapter(
+                arrayListOf(),
+                R.layout.badge_list_rv_item
+            )
+        observeBadgeList()
+        recyclerViewBadgeList.adapter = recyclerViewBadgeListAdapter
+
+        layoutManager = LinearLayoutManager(this)
+        recyclerViewBadgeList.layoutManager = layoutManager
+    }
+
+    private fun observeBadgeList() {
+        MockBadgeListViewModel.createMockedBadgeListLiveData().observe(this, Observer {
```

Comment:
Based on my previous comment: why don't you have a BadgeListViewModel and instantiate it normally and just call it here? Something like: `badgeListViewModel.getBadges()`

Reply:
Same as the above comment. @viniciusalbuquerque